server/models/Pet.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+const User = require('./User');
+const Appointment require('./Appointment');
+
+const petSchema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  birthday: {
+    type: Integer,
+    required: true
+  },
+  pet: {
```

Comment:
we should not define a column/property named pet on the collection name Pet. This column/property should be name.

Reply:
The pet property was the one we agreed upon to have the pet type (Dog, Cat, etc), the field for the name is already there as well, you can see that in the wireframes.

Reply:
I can change it to petType if you prefer

Reply:
The pet schema should have a property for the pet's name.

Reply:
Pricisila, the name is included in the schema, see below, is the first property that I added to it.

const petSchema = new Schema({
  name: {
    type: String,
    required: true

Reply:
I can call it petName, if you want